docs/excel/custom-functions-runtime.md

```diff
@@ -0,0 +1,149 @@
+---
+ms.date: 09/18/2018
+description: Excel custom functions use a new JavaScript runtime, which differs from the standard Add-ins WebView control runtime.
+title: Runtime for Excel Custom Function Add-ins
+---
+
+# Runtime for Excel custom function add-ins
+
+Custom functions extend Excel’s capabilities using a new JavaScript runtime. This runtime utilizes a sandboxed JavaScript engine rather than a web browser. Additionally, it prioritizes performance, allowing you to run thousands of custom functions simultaneously.  
+
+> [NOTE!] The code for your add-in may include other parts, such as task panes and other UI elements.  
```

Comment:
Note is not formatted correctly.   See the [staging site](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-runtime?branch=michelle-runtime)

should be:  > [!NOTE]